CMakeLists.txt

```diff
@@ -6,48 +6,52 @@
 
 cmake_minimum_required(VERSION 3.3)
 
+# If we are not building as a part of LLVM, build Flang as a
+# standalone project, using LLVM as an external library:
+if( CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR )
+  project(Flang)
+endif()
```

Comment:
If this is absolutely needed here, near the top of the file, then I suggest removing lines 59-60.

Reply:
Done in a4c90f89